schemas/io-package.json

```diff
@@ -0,0 +1,457 @@
+                    "type": "object",
+                    "required": ["en"],
+                    "patternProperties": {
+                        "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                            "type": "string"
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "news": {
+                    "description": "Short form of changelog in all supported languages",
+                    "type": "object",
+                    "minProperties": 1,
+                    "maxProperties": 20,
+                    "patternProperties": {
+                        "^\\d+\\.\\d+\\.\\d+.*": {
+                            "type": "object",
+                            "required": ["en"],
+                            "patternProperties": {
+                                "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                                    "type": "string"
+                                }
+                            },
+                            "additionalProperties": false
+                        }
+                     },
+                    "additionalProperties": false
+                },
+                "desc": {
+                    "description": "Description, object for multilingual description",
+                    "type": "object",
+                    "required": ["en"],
+                    "patternProperties": {
+                        "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                            "type": "string"
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "messagebox": {
+                    "description": "true if messagebox supported. If yes, the object system.adapter.<instance>.messagebox will be created to send messages to adapter",
+                    "type": "boolean"
+                },
+                "readme": {
+                    "description": "[DEPRECATED]: Use docs",
+                    "type": "string"
+                },
+                "docs": {
+                    "description": "The structure like {\"en\": \"docs/en/README.md\", \"de\": [\"docs/de/README.md\", \"docs/de/README1.md\"]} that describes the documentation if not in README.md",
+                    "type": "object",
+                    "required": ["en"],
+                    "patternProperties": {
+                        "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                            "type": "string"
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "authors": {
+                    "description": "Array of authors as string",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
```

Comment:
Do we have a minimum number of authors?

Reply:
We could say one if the property is given, according to docs its optional, and thus it is not required in my schema definition

Reply:
that should work